src/core/definitions.jl

```diff
@@ -65,6 +65,7 @@ const SECONDS_IN_HOUR = 3600.0
 const MAX_START_STAGES = 3
 const OBJECTIVE_FUNCTION_POSITIVE = 1.0
 const OBJECTIVE_FUNCTION_NEGATIVE = -1.0
+const DEFAULT_RESERVE_COST = 1.0e-4
```

Comment:
Add a comment about the use of this constant. e.g. This value is used to avid degeneracy of the solutions.